MAINTAINERS.md

```diff
@@ -0,0 +1,105 @@
+# Maintainers
+
+## Maintainer Scopes, GitHub Roles and GitHub Teams
+
+Maintainers are assigned the following scopes in this repository:
+
+| Scope      | Definition               | GitHub Role | GitHub Team                          |
+| ---------- | ------------------------ | ----------- | ------------------------------------ |
+| Admin      |                          | Admin       | [aries-admins]                       |
+| Maintainer | The GitHub Maintain role | Maintain    | [aries-cloudagent-python committers] |
+| Triage     | The GitHub Triage role   | Triage      | [aries triage]                       |
+| Read       | The GitHub Read role     | Read        | [Aries Contributors]                 |
+
+[aries-admins]: https://github.com/orgs/hyperledger/teams/aries-admins
+[aries-cloudagent-python committers]: https://github.com/orgs/hyperledger/teams/aries-cloudagent-python-committers
+[aries triage]: https://github.com/orgs/hyperledger/teams/aries-triage
+[Aries Contributors]: https://github.com/orgs/hyperledger/teams/aries-contributors
+
+## Active Maintainers
+
+<!-- Please keep this sorted alphabetically by github -->
+
+| GitHub ID       | Name          | Scope | LFID | Discord ID | Email                  | Company Affiliation |
+| --------------- | ------------- | ----- | ---- | ---------- | ---------------------- | ------------------- |
```

Comment:
This list is not accurate — sorry, I should have checked the teams first.  Should be the same list as ACA-Py, I think. I can see the teams and verify.  I’ll provide later today.

Reply:
Updated